packages/storybook/stories/Localization.stories.mdx

```diff
@@ -8,3 +21,86 @@ Localization is a key to making products that can be used across the world and b
 UI Library will provide out of the box support for some languages and capabilities such as RTL.
 Developers can provide their own localization files to be used for the UI Library.
 These localization capabilities will be added ahead of General Availability.
+
+## LocalizationProvider
+
+`LocalizationProvider` is a wrapping component that sets all the strings for ACS components using a `Locale`. We have a set of
+locales available for import to use. LocalizationProvider is not required as all components default to using our English (US) locale.
+
+### Usage
+
+To use the LocalizationProvider, simply import it from `@azure/communication-react` and wrap it around ACS UI SDK components,
+For the example below we will localize buttons in a `ControlBar` in French (France) and import that locale `fr_FR` from `@azure/communication-react`.
+
+<Source code={LocalizationProviderSnippetText} />
+
+<Canvas withSource="none">
+  <LocalizationProviderSnippet />
+</Canvas>
+
+## Overriding our locales
```

Comment:
nit: Drop 'our'.

Alos, how about "Customizing locales". Override seems to imply that you're replacing the whole thing.